src/components/Combobox/components/ComboboxCore.js

```diff
@@ -18,6 +18,10 @@ class ComboboxCore extends Component {
      * Classname that will be applied to the <FormElement> wrapping input and listbox
      */
     comboboxClassName: PropTypes.string,
+    /**
```

Comment:
Can you add a note to all three props and note that these are custom additions to the LDS combobox specs?

Reply:
👍